app/controllers/users/sessions_controller.rb

```diff
@@ -53,6 +50,11 @@ def check_user_needs_redirect
       end
     end
 
+    def process_locked_out_user
+      render 'two_factor_authentication/shared/max_login_attempts_reached'
+      sign_out
```

Comment:
Not sure. I only refactored this to please Code Climate. I didn't change the behavior.